notebooks/code_samples/nlp_and_llm/foundation_models_integration_demo.ipynb

```diff
@@ -236,10 +236,7 @@
    "source": [
     "import pandas as pd\n",
     "\n",
-    "df = pd.read_csv(\"./datasets/sentiments.csv\")\n",
-    "\n",
-    "df_test = df[:10].reset_index(drop=True)\n",
-    "df_test"
+    "df = pd.read_csv(\"./datasets/sentiments_with_predictions.csv\")"
```

Comment:
We should create a python class to load dataset similar to what we do for the customer churn dataset. Something like this : 
`from validmind.datasets.classification import cnn_dailymail`
Let's create a ticket for this if we are not doing this in PR.